### ISSUE TITLE
[F-Req] "are you sure?" Message request

### ISSUE THREAD

**Issue by TotallyNotSaeed** [PojavLauncherTeam/PojavLauncher#5625]
### Is this request related to a bug?

no

### Describe the request.

There should be a message that says "are you sure you want to delete this profile" when deleting a profile

![image](https://github.com/PojavLauncherTeam/PojavLauncher/assets/158475307/ec475ccd-299e-4188-93e6-0725dc8d91cf)

### Anything else?

I sometimes mistakenly hit "delete" when trying to save

**Comment by Mathias-Boulay** [comment]
yeah, I should also inverse save and delete positions while I'm at it due to design inconsistencies
